giraffe/src/types/index.ts

```diff
@@ -29,7 +29,8 @@ export interface Config {
   xScale?: string
   yScale?: string
 
-  table: Table
+  table?: Table
```

Comment:
i *think* i understand what's going on here, but i just want to make sure. [elsewhere in the pr](https://github.com/influxdata/giraffe/pull/218/files#diff-41549c804c9f3592a0ab56556e0096e9R113) there's an unsafe accesses to `table` - they assume that `table` will always exist. that is a slightly different data structure with a `table` property that is required, right?

Reply:
Yes. That unsafe access is on a different interface. The result of calling `fromFlux` is an object defined by the  `FromFluxResult` interface, which _always_ has a table property.